### PR TITLE
57 - UDT support

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

File: ext/README.md [hunk 1]
```diff
@@ -112,13 +112,12 @@ Apache Cassandra.
...
-simplify the build process on Windows. If you have more than one version of
-Visual Studio installed you will be prompted to which version to use when
-compiling the driver.
+simplify the build process on Windows and select the correct version of Visual
+Studio for the PHP version the driver is being built for.
 
 First you will need to open a "Command Prompt" to execute the batch script.
 Running the batch script without any arguments will build the driver for PHP
-v5.6 with Zend thread safety (ZTS) and for the current system architecture
+v7.0 with Zend thread safety (ZTS) and for the current system architecture
```
Comment (jippi) [inline ext/README.md]: dropping 5.6 ?

File: ext/doc/Cassandra.php [hunk 2]
```diff
@@ -322,7 +322,12 @@
     /**
      * Current version of the extension.
      */
-    const VERSION = '1.0.0-beta';
+    const VERSION = '1.1';
+
+    /**
+     * Version of the cpp-driver the extension is compiled against.
+     */
+    const CPP_DRIVER_VERSION = '2.2.2';
```
Comment (jippi) [inline ext/doc/Cassandra.php:330]: could this be exposed in phpinfo() as well?

**Comment by mpenick** [comment]
@jippi The cpp-driver version is now exposed via phpinfo(): https://github.com/datastax/php-driver/blob/57/ext/php_cassandra.c#L415-L418

**Comment by mpenick** [comment]
@jippi Supporting both PHP 5.6 and 7.0.

**Comment by mpenick** [comment]
I think that "UserType" and "UserValue" are better names than "Udt". Considering changing.

**Comment by jippi** [comment]
@mpenick I agree!

**Comment by mpenick** [comment]
Merge into https://github.com/datastax/php-driver/pull/60

**Pull request closed without merging**